editors/code/src/tasks.ts

```diff
@@ -117,7 +140,27 @@ export async function buildCargoTask(
         name,
         TASK_SOURCE,
         exec,
-        ['$rustc']
+        ["$rustc"]
+    );
+}
+
+export async function buildWasmTask(
+    scope: vscode.WorkspaceFolder | vscode.TaskScope | undefined,
+    definition: CargoTaskDefinition,
+    name: string,
+    args: string[]
+): Promise<vscode.Task> {
+    const exec = new vscode.ProcessExecution("wasmer", args, definition);
```

Comment:
I think this should be configurable without a default runner instead of forcing wasmer.

Reply:
Yes you're right ! I planned to do this